spec/models/concerns/full_text_searchable_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of hitobito and licensed under the
+#  Affero General Public License version 3 or later. See the COPYING file at the top-level directory
+#  or at https://github.com/hitobito/hitobito.
+
+require "spec_helper"
+
+describe FullTextSearchable do
```

Comment:
Bisher haben wir alle Fulltext Search Specs hier gemacht, da wir so unsere eigentliche Suche absetzen
https://github.com/hitobito/hitobito/tree/master/spec/domain/search_strategies

Finde die Specs hier nicht falsch (besonders weil das Problem direkt mit dem Query verbunden ist), aber eventuell wäre es besser alle Specs and einem Ort zu haben.

Reply:
Mhh, finde ich unproblematisch ..